Packages/ClientRuntime/Sources/Config/DefaultSDKRuntimeConfiguration.swift

```diff
@@ -6,12 +6,25 @@
 //
 
 public struct DefaultSDKRuntimeConfiguration: SDKRuntimeConfiguration {
```

Comment:
We should rename this struct and the protocol to something more descriptive and without "SDK" in the name.
I'll file an issue to address that